config/msb_env.sh

```diff
@@ -2,7 +2,7 @@
 
 export MSB="${HOME}/motion-sensor-box"
 export MSB_SRC="${MSB}/src"
-export MSB_CONFIG_DIR="${HOME}/.config/"
+export MSB_CONFIG_DIR="${HOME}/.config/msb/"
```

Comment:
This should be `MSB_CONFIG_DIR="${HOME}/.config/msb/conf.d"`

Reply:
moved the `msb_env.sh` to conf/msb/ and fixed it